src/components/CircularProgressBar/CircularProgressBar.async.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
```

Comment:
maybe instead of introducing a new test file, it can be another `describe` in the original spec?

Reply:
We have some problems with `wix-ui-test-utils/driver-factory`. For some reason, it isn't unmounting component from the virtual dom, so after one test for tooltip presence was passed, the second one is failing, when checking that tooltip is not present before component was loaded. Jest is running it in separate scopes with own JSDOM environment. I wanted to dive deeper into it and fix in separate PR and for now, have async stuff as a separate file.

Reply:
did you close the tooltip between tests? are you waiting for it to disappear?

Reply:
But according to the Tooltip tests, it isn't gone even after `mouseLeave` action
https://github.com/wix/wix-ui-backoffice/blob/master/src/components/Tooltip/Tooltip.spec.tsx#L22